src/families/bitcoin/bridge/libcore.js

```diff
@@ -71,7 +71,13 @@ const createTransaction = () => ({
   useAllAmount: false,
 });
 
-const updateTransaction = (t, patch) => ({ ...t, ...patch });
+const updateTransaction = (t, patch) => {
+  const updatedT = { ...t, ...patch };
+  if (updatedT.recipient.toLowerCase().indexOf("bc1") === 0) {
+    updatedT.recipient = updatedT.recipient.toLowerCase();
+  }
+  return updatedT;
+};
```

Comment:
to test this effectively work, can you modify one of the test-dataset.js file to change one of the test using an upper case address when it's used to validate a tx? thanks